assets/wizards/subscriptionsOnboarding/index.js

```diff
@@ -58,288 +64,180 @@ class SubscriptionsOnboardingWizard extends Component {
-						/>
-						<Button
-							className='is-centered'
-							isTertiary
-							href={ newspack_urls['checklists']['memberships'] }
-						>
-							{ __( 'Back to checklist') }
-						</Button>
-					</Card>
-				</Fragment>
-			);
-		}
-
-		if ( 1 === wizardStep ) {
-			return (
-				<Fragment>
-					{ error }
-					<LocationSetup
-						countrystateFields={ fields.countrystate }
-						currencyFields={ fields.currency }
-						location={ location }
-						onChange={ location => this.setState( { location } ) }
-						onClickContinue={ () => this.saveLocation() }
+		const { pluginRequirements } = this.props;
+		const { location, stripeSettings, fields } = this.state;
+		return (
+			<HashRouter hashType="slash">
+				<Switch>
+					{ pluginRequirements }
+					<Route
+						path="/"
+						exact
+						render={ routeProps => (
+							<Fragment>
+								<FormattedHeader
+									headerText={ __( 'About your publication' ) }
+									subHeaderText={ __( 'This information is required for accepting payments' ) }
+								/>
+								<LocationSetup
+									countrystateFields={ fields.countrystate }
+									currencyFields={ fields.currency }
+									location={ location }
+									onChange={ location => this.setState( { location } ) }
+									buttonText={ __( 'Save' ) }
+									buttonAction={ () =>
+										this.saveLocation().then(
+											() => routeProps.history.push( '/stripe' ),
+											() => null
+										)
+									}
+								/>
+							</Fragment>
+						) }
 					/>
-				</Fragment>
-			);
-		}
-
-		if ( 2 === wizardStep ) {
-			return (
-				<Fragment>
-					{ error }
-					<PaymentSetup
-						stripeSettings={ stripeSettings }
-						onChange={ stripeSettings => this.setState( { stripeSettings } ) }
-						onClickFinish={ () => this.saveStripeSettings() }
+					<Route
+						path="/stripe"
+						render={ routeProps => (
+							<Fragment>
+								<FormattedHeader
+									headerText={ __( 'Set up Stripe' ) }
+									subHeaderText={ __( 'Stripe is the recommended gateway for accepting payments' ) }
+								/>
+								<PaymentSetup
+									stripeSettings={ stripeSettings }
+									onChange={ stripeSettings => this.setState( { stripeSettings } ) }
+									buttonText={ __( 'Save' ) }
+									buttonAction={ () =>
+										this.saveStripeSettings().then(
+											() => ( window.location = newspack_urls[ 'checklists' ][ 'memberships' ] )
+										)
+									}
+								/>
+							</Fragment>
+						) }
 					/>
-				</Fragment>
-			);
-		}
-
-		// Redirect to Memberships checklist if all steps are complete.
-		window.location = newspack_urls['checklists']['memberships'];
-		return null;
+					<Redirect to="/" />
+				</Switch>
+			</HashRouter>
+		);
 	}
 }
+
 render(
-	<SubscriptionsOnboardingWizard />,
+	createElement( withWizard( SubscriptionsOnboardingWizard), {
+		buttonText: __( 'Back to checklist' ),
+		buttonAction: newspack_urls[ 'checklists' ][ 'memberships' ],
+	} ),
```

Comment:
[This wizard has a requirement of the WooCommerce plugin](https://github.com/Automattic/newspack-plugin/pull/104/files#diff-07e22c86f90e246a09451508cbf0db5eR20), otherwise we get fatal errors as soon as the wizard is loaded:

> Unrecoverable error
> The WooCommerce plugin is not installed and activated. Install and/or activate it to access this feature.

Reply:
Oops, don't know how that went missing... Fixed in https://github.com/Automattic/newspack-plugin/pull/109/commits/da38f2a2596cedc67477eca70c7d47b4d7bd67fd